.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

```suggestion
blank_issues_enabled: true
```

I think this should be true.